web/src/composebox_typeahead.ts

```diff
@@ -79,9 +79,13 @@ export type LanguageSuggestion = {
     type: "syntax";
 };
 
-type TopicSuggestion = {
+export type TopicSuggestion = {
     topic: string;
     type: "topic_list";
+    // This will be used when we want to only render the stream as an option in the topic typeahead
+    // while having #**stream_name> as the token.
+    is_channel_link: boolean;
+    stream_data: StreamPillData;
```

Comment:
Just to be clear, is it `is_channel_link`, `stream_data`, or both that the "this" applies to? 

It probably is worth redrafting a bit to make it as explicit as possible as to what is being described.

Reply:
Similarly, this updated comment should be in the first commit.